channeldb/invoices.go

```diff
@@ -1905,9 +1909,7 @@ func (d *DB) updateInvoice(hash *lntypes.Hash, refSetID *invpkg.SetID, invoices,
 
 	now := d.clock.Now()
 
-	invoiceIsAMP := invoiceCopy.Terms.Features.HasFeature(
-		lnwire.AMPOptional,
-	)
+	invoiceIsAMP := invoiceCopy.IsAMP()
```

Comment:
Now that we've got a method, could move this down inline to 1960 rather than creating a specific bool.

Reply:
It is checked in 7 more places in this function. I guess that now that it is shorter to write we can call the function instead of caching the result.

Reply:
Ah, that's what I get for not looking past the GH diff :')